pkg/streamingpromql/README_FUZZ_TESTING.md

```diff
@@ -0,0 +1,137 @@
+# Query Fuzz Testing
+
+This file contains notes related to the addition of Fuzz tests for the Mimir query engine.
+
+Fuzz tests have been added to the `engine_fuzz_test.go` file. 
+
+These will run as a normal unit test, or they can be invoked with the Go Fuzz engine with `go test -fuzz=FuzzQuery`
+
+In either case, the test function loads a set of sample data from `seed-data.test` and then runs a range of sample queries from `seed-queries.test`.
```

Comment:
I would avoid documenting this and instead rely on the code to explain this (otherwise it'll get out-of-sync too easily).